taxonomy-editor-frontend/src/pages/home/index.tsx

```diff
@@ -8,7 +8,11 @@ import Typography from "@mui/material/Typography";
 import Stack from "@mui/material/Stack";
 import Container from "@mui/material/Container";
 
-const Home = ({ resetNavLinks }) => {
+type Props = {
+  resetNavLinks: () => void;
+};
+
+const Home = ({ resetNavLinks }: Props) => {
```

Comment:
Shouldn't this be done for [other files](https://github.com/openfoodfacts/taxonomy-editor/search?q=resetnavlinks) where resetNavLinks is used too?